addons/project_timesheet_holidays/models/hr_holidays.py

```diff
@@ -135,7 +135,16 @@ def action_refuse(self):
     def _action_user_cancel(self, reason):
         res = super()._action_user_cancel(reason)
         timesheets = self.sudo().timesheet_ids
-        timesheets.write({'holiday_id': False})
+        timesheets.holiday_id = False
```

Comment:
It's better to avoid unnecessary diff in stable

Reply:
Thank you for the review Vincent, I will revert the change.